### DISCUSSION TITLE
Recommendation for passwordless registration/log in

### DISCUSSION THREAD

**Discussion by thammarith** [directus/directus#15850] [Archive]
Hi, I'm exploring if it is possible for Directus to do passwordless registeration/log in?

If I want to build a PoC _without_ using 3rd party authentication (such as AWS Cognito) to avoid vendor-lock and keep the user data inside Directus, what should be the architecture? Is that possible with the latest Directus?

The flow that I'm having in mind is user registers with mobile/email address --> an OTP is sent to the mobile number/email address --> user enters the OTP --> logs the user in. However, I'm not sure how to implement that on Directus.

Thanks!

**Comment by rijkvanzanten** [comment]
There's currently no native "local" "magic link" login (make sure to upvote https://github.com/directus/directus/discussions/3068). That being said, you could setup auth0 or another OpenID provider, and require that to use magic sign in to have users log into Directus without a password.
